client.go

```diff
@@ -128,7 +130,14 @@ func (d *aivenClient) do(ctx context.Context, method, path string, v any) (*http
 	}
 
 	req.Header.Set("Content-Type", "application/json")
-	req.Header.Set("User-Agent", d.UserAgent)
+	req.Header.Set(
+		"User-Agent",
+		strings.TrimSpace(fmt.Sprintf(
+			"go-client-codegen/%s %s",
```

Comment:
```suggestion
			"go-client/%s/%s",
```
I think, it shouldn't have spaces.

Reply:
It should have spaces as per the RFC. The final User-Agent is going to look like: `go-client/1.0.0 terraform-provider-aiven/1.0.0`.